src/components/ConnectedFilter/index.jsx

```diff
@@ -170,7 +174,7 @@ class ConnectedFilter extends React.Component {
         className={this.props.className}
         tabs={filterTabs}
         filterConfig={this.props.filterConfig}
-        onFilterChange={(e) => this.handleFilterChange(e)}
+        onFilterChange={e => this.handleFilterChange(e)}
```

Comment:
i think we have different eslint rules? The default eslint `arrow-parens` rule should be `"always"`, which is discouraging of removing these parentheses

Reply:
i ran `npm run eslint` and it removed them 🤔